internal/provider/cluster_resource.go

```diff
@@ -247,6 +247,11 @@ func (r *clusterResource) Schema(
 					validators.FolderParentID(),
 				},
 			},
+			"delete_protection": schema.BoolAttribute{
+				Computed:    true,
+				Optional:    true,
+				Description: "Set to true to enable delete protection on the cluster.",
```

Comment:
It feels like the behavior of not setting it should also be documented somewhere.  Specifically how the behavior interacts with a user who has set the value via the UI.

Reply:
I updated this description to describe the behavior. Let me know what you think.

Reply:
It seems truthful.  I'm still a bit concerned about the usability of this documentation but I'm good with this for now. We can revise it later after discussion with @marksoper and the docs team.  I know they're already planning a revamp of the docs. 

To be more specific, It seems like the logical next question is to ask is "well... what does the server set it to?".  If it were me, I'd rather just set it at that point than go check what the actual default is.